pair/tests/utils.rs

```diff
@@ -1,5 +1,5 @@
 use dex_factory_io::*;
-use dex_pair_io::TokenId;
+use dex_pair_io::FungibleId;
```

Comment:
```suggestion
```
It's already imported by `use dex_pair_io::*;` on the line below.

Reply:
We should add it explicitly, otherwise it's an ambiguous import.